model/flow/dkg.go

```diff
@@ -14,9 +14,9 @@ const (
 	DKGEndStateNoKey
 	// DKGEndStateDKGFailure - the underlying DKG library reported an error.
 	DKGEndStateDKGFailure
-	// DKGEndStateRecovered - this node has recovered its beacon key from a previous epoch.
-	// This occurs only for epochs which are entered through the EFM Recovery process ([flow.EpochRecover] service event).
-	DKGEndStateRecovered
+	// RandomBeaconKeyRecovered - this node has recovered its beacon key from a previous epoch.
+	// This occurs only for epochs which are entered through the EFM Recovery process (`flow.EpochRecover` service event).
```

Comment:
Minor point on the formatting:
- Wrapping code references in `[]` [formats it as a link to the type in Godoc](https://go.dev/doc/comment#doclinks)
- Wrapping code references in ``` `` ``` doesn't have any formatting support (Markdown-style code spans are unsupported).

Reply:
Good to know, thanks. I will make a PR to `feature/efm-recovery` to fix any of those.